### PR TITLE
refresh events on sigstore website with newer events.

### PR DESCRIPTION
**Pull request by jonvnadelberg**
Fix for issue #276 

#### Summary

Events were out out date on website, and did not include important recent events.  this pr places them on website.
#### Release Note

NONE
#### Documentation

doc only

### CODE REVIEW

**Comment by jonvnadelberg** [comment]
@haydentherapper @ltagliaferri  Please review added events.

File: content/events/sigstore-npm-beta.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: Sigstore npm public beta released
```
Comment (haydentherapper) [inline content/events/sigstore-npm-beta.md:2]:
```suggestion
title: Sigstore support in npm released in public beta
```

File: content/events/sigstore-python-1-0.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+title: Sigstore-Python 1.0 released
```
Comment (haydentherapper) [inline content/events/sigstore-python-1-0.md:2]:
```suggestion
title: sigstore-python 1.0 released
```

**Review by haydentherapper — APPROVED** [review]

File: content/events/cosign-2-0-launch.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+title: Cosign 2.0 released
+pubDate: 20233-02-23
```
Comment (ltagliaferri) [inline content/events/cosign-2-0-launch.md:3]:
```suggestion
pubDate: 2023-02-23
```

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Just a date typo here

**Comment by jonvnadelberg** [comment]
@ltagliaferri date typo fixed on pr #277 can we merge?

**Comment by jonvnadelberg** [comment]
Hi @ltagliaferri can this be merged now?   It looks to be done.

**Review by ltagliaferri — APPROVED** [review]

**Pull request merged**